### PR TITLE
Update EIP-2015: Move to Review

### PR DESCRIPTION
**Pull request by Pandapip1**
Needs review from either Pedro Gomes (@pedrouid) or Erik Marks (@rekmarks).

There are also a few more changes I will need to make, particularly formatting/grammar-wise

### CODE REVIEW

**Comment by Pandapip1** [comment]
CC Pedro Gomes (@pedrouid) or Erik Marks (@rekmarks)

**Comment by Pandapip1** [comment]
CC again Pedro Gomes (@pedrouid) or Erik Marks (@rekmarks)

**Comment by Pandapip1** [comment]
CC again again Pedro Gomes (@pedrouid) or Erik Marks (@rekmarks)

**Review by pedrouid — DISMISSED** [review]
👍

**Comment by Pandapip1** [comment]
@eth-bot rerun

File: EIPS/eip-2015.md [hunk 1]
```diff
@@ -1,108 +1,74 @@
...
+  chainId: string;
+  chainName?: string;
+  rpcUrls?: string[];
+  nativeCurrency?: NativeCurrencyData;
+  blockExplorerUrl?: string;
 }
 ```
 
-### Responses
+The `chainId` is the `0x`-prefixed [EIP-155](./eip-155.md)-compliant chain ID. The `chainName` is a suggest human-readable name of the chain, to be displayed to the user. The `rpcUrls` array is a list of RPC endpoints for the given `chainId`. The `nativeCurrency` object suggests how the native currency should be displayed. Its parameters, `name`, `symbol`, and `decimals`, should be interpreted like in [ERC-20](./eip-20.md). Finally, the `blockExplorerUrl` should link to a block explorer compatible with the given `chainId`.
```
Comment (SamWilsn) [inline EIPS/eip-2015.md:46]:
```suggestion
The `chainId` is the `0x`-prefixed [EIP-155](./eip-155.md)-compliant chain ID. The `chainName` is a suggested human-readable name of the chain, to be displayed to the user. The `rpcUrls` array is a list of RPC endpoints for the given `chainId`. The `nativeCurrency` object suggests how the native currency should be displayed. Its parameters, `name`, `symbol`, and `decimals`, should be interpreted like in [ERC-20](./eip-20.md). Finally, the `blockExplorerUrl` should link to a block explorer compatible with the given `chainId`.
```

File: EIPS/eip-2015.md [hunk 2]
```diff
@@ -1,108 +1,74 @@
...
+
+The `wallet_updateEthereumChain` method is namespaced under `wallet_` to avoid conflicts with other methods. The `wallet_` prefix is used by other methods that are wallet-specific, such as `wallet_addEthereumChain` and `wallet_switchEthereumChain`.
+
+## Backwards Compatibility
+
+This EIP is fully backwards compatible.
+
+## Security Considerations
+
+### SSRF
```
Comment (SamWilsn) [inline EIPS/eip-2015.md:64]:
```suggestion
### Server-Side Request Forgery (SSRF)
```

**Review by SamWilsn — APPROVED** [review]
The changes since @pedrouid reviewed are minimal. I'm going to override the bot here.

**Pull request merged**